mlflow/sklearn/utils.py

```diff
@@ -310,7 +311,10 @@ def plot_confusion_matrix(*args, **kwargs):
                 "axes.labelsize": 10.0,
             }
         ):
-            return sklearn.metrics.plot_confusion_matrix(*args, **kwargs)
+            if Version(sklearn.__version__) >= Version("1.0"):
```

Comment:
```suggestion
            if Version(sklearn.__version__) >= Version("1.0"):
                from sklearn.metrics import ConfusionMatrixDisplay
```

Can we import `ConfusionMatrixDisplay` here?